lib/common/utils/graphql.dart

```diff
@@ -237,6 +237,48 @@ abstract class GqlQuery {
     }
   ''';
 
+  static const schedule = r'''
```

Comment:
I think a lot of data can be dropped from the query - we don't need `season`, `duration`, `isAdult`... or just anything that isn't on the tile. Plus, `airingAt`, `timeUntilAiring` and `episode` seem to be duplicated? And do we need both `airingAt` and `timeUntilAiring`? What field you could add, is the current progress of the viewer.